src/app/lib/config/optimizely/index.js

```diff
@@ -1,11 +1,18 @@
 export default {
   // This is the key of the whole experiment
-  flagKey: 'newswb_ws_topbarojs',
+  flagKey: 'newswb_ws_litesitepromotion',
   // This is the key for the 'rule' that is attached to the experiment
-  ruleKey: 'newswb_ws_topbarojs',
-  viewClickAttributeId: 'wsoj',
+  ruleKey: 'newswb_ws_litesitepromotion',
+  viewClickAttributeId: 'lite-site-cta',
```

Comment:
Is this the right value for this? Or should it be `canonical-lite-cta`?

Reply:
I think it does need to be `canonical-lite-cta` because `lite-site-cta` is this one: https://github.com/bbc/simorgh/blob/latest/src/app/components/LiteSiteCta/index.tsx#L35

Whereas we're updating https://github.com/bbc/simorgh/blob/WORLDSERVICE-418-lite-cta-optimizely/src/app/pages/ArticlePage/ArticleHeadline/index.tsx#L22

```suggestion
  viewClickAttributeId: 'canonical-lite-cta',
```